Update destination-event-streaming-overview.md (#129)
amplitude/amplitude-dev-center: docs/data/destination-event-streaming-overview.md

description: Send Amplitude data to other tools in your stack with just a few clicks, using no-code event streaming integrations. 
---

> Event Streaming empowers you to share your Amplitude data seamlessly throughout your entire system. It allows you to utilize the valuable behavioral data in Amplitude to enhance customer profiles and send data to your marketing, sales, and infrastructure tools.

Reviewer: ```suggestion
Event Streaming lets you share your Amplitude data throughout your entire system. Use the valuable behavioral data in Amplitude to enhance customer profiles and send data to your marketing, sales, and infrastructure tools.
```

@Brandoncharleskhoo1 What "sytstem" are you talking about here?

> With event streaming, you gain access to user-friendly, configuration-based tools that offer precise control over the data you transmit. You can filter data by user, group, and event properties, ensuring that only the relevant information is delivered to your downstream tools. Additionally, you can monitor key metrics such as event volume, latency, and detailed delivery status to assess the performance and reliability of your streaming integration.

Reviewer: ```suggestion
With event streaming, you gain access to user-friendly, configuration-based tools that offer precise control over the data you send. Filter data by user, group, and event properties, ensuring that you send only the relevant information to your downstream tools. Additionally, you can monitor key metrics like event volume, latency, and detailed delivery status to assess the performance and reliability of your streaming integration.
```

## Considerations

> - **Billing efficiency:** Amplitude tracks event volume based on distinct events sent. If the same event is sent to multiple Event Streaming destinations, it's counted only once for billing.

Reviewer: ```suggestion
- **Billing efficiency:** Amplitude tracks event volume based on distinct events sent. If you send same event to multiple Event Streaming destinations, it's counted only once for billing.
```

- **Latency target:** Amplitude aims for an end-to-end p95 latency of 60 seconds, monitored and supported by alerts.

> - **Retry mechanism:** Intermittent errors are addressed through in-memory retries with exponential backoff for initial sends. A robust retry pipeline attempts up to 10 times within a 4-hour window. This mechanism applies to all Event Streaming destinations.

Reviewer: ```suggestion
- **Retry mechanism:** Amplitude addresses Intermittent errors using in-memory retries with exponential backoff for initial sends. The retry pipeline attempts up to 10 times within a 4-hour window. This mechanism applies to all Event Streaming destinations.
```

> - **Streamlined monitoring and management:** The Event Streaming Debugger UI in Amplitude Data enables users to monitor pending retries, progress, and expired payloads. Insights into error categories and failed payload samples are provided for analysis.

Reviewer: ```suggestion
- **Streamlined monitoring and management:** The Event Streaming Debugger UI in Amplitude Data lets you monitor pending retries, progress, and expired payloads. Analyze failed payload samples to gain insight into error categories.
```
